server.js

```diff
@@ -1,32 +1,51 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import booksData from './data/books.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
+const ERROR_BOOKS_NOT_FOUND = {error: "Sorry, we couldn't find any books matching your search"}
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Routes
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to Books API, type in /books for an API of all the books we have in the library')
 })
 
+app.get('/books', (req, res) => {
+  const {authors, title} = req.query;
```

Comment:
Maybe this could be called `author` since you often make a query on a single thing

Reply:
Some spacing between is nice -> `{ author, title }`